pipelines/prep_data.py

```diff
@@ -448,7 +461,21 @@ def process_and_save_state(
         "nhsn_step_size": nhsn_step_size,
         "state_pop": state_pop,
         "right_truncation_offset": right_truncation_offset,
+        "data_observed_disease_wastewater": preprocessed_ww_data.select(
+            [
+                "date",
+                "site",
+                "lab",
+                "site_pop",
+                "site_index",
+                "lab_site_index",
+                "log_genome_copies_per_ml",
+                "log_lod",
+                "below_lod",
+            ]
+        ),
```

Comment:
Any reason for this not to be what is returned by `preprocess_ww_data` so we don't have to manipulate it afterword?